.golangci.yml

```diff
@@ -17,47 +17,45 @@ linters:
     - contextcheck
     - decorder
     - durationcheck
+    - errcheck
     - errchkjson
     - errname
-    - errcheck
     - errorlint
     - exhaustive
-    - exportloopref
```

Comment:
why are you removing linters here?

Reply:
```sh
❯ gco main
Switched to branch 'main'                  
Your branch is up to date with 'upstream/main'.
❯ gl
Already up to date.
❯ golangci-lint run
WARN [config_reader] The configuration option `linters.errcheck.ignore` is deprecated, please use `linters.errcheck.exclude-functions`. 
WARN [lintersdb] The name "vet" is deprecated. The linter has been renamed to: govet. 
WARN [lintersdb] The name "vetshadow" is deprecated. The linter has been renamed to: govet. 
WARN The linter 'exportloopref' is deprecated (since v1.60.2) due to: Since Go1.22 (loopvar) this linter is no longer relevant. Replaced by copyloopvar. 
ERRO [linters_context] exportloopref: This linter is fully inactivated: it will not produce any reports. 
```